src/chrome/buildContextMenus.js

```diff
@@ -0,0 +1,100 @@
+import { setOptions } from '../actions';
+import {
+  CONFIRM_WORK_ASSIGNMENT,
+  NOTIFICATION_SOUND_URL,
+  NOTIFICATION_SOUND_REPEAT,
+} from '../constants';
+
+const buildContextMenus = (store, chrome) => {
+  const pluginOptions = store.getState().plugin.get('options');
+
+  const isChecked = (name, value) => (
+    pluginOptions.get(name) === value
+  );
+
+  const onMenuClick = (name, value) => {
+    store.dispatch(setOptions({ [name]: value }));
+
+    chrome.storage.sync.set({
+      options: store.getState().plugin.get('options').toJS(),
+    });
+  };
+
+  const createContextMenu = (option) => {
```

Comment:
There will be a period between when the backend JS is loaded and people's plugins have updated when the extension won't have the `contextMenus` privilege. I checked and it looks like `chrome.contextMenus` is undefined in that case, so this will blow up. Can you co a check and maybe just return if `chrome.contexteMenus` is undefined?